azurerm/resource_arm_subnet.go

```diff
@@ -90,14 +90,18 @@ func resourceArmSubnet() *schema.Resource {
 										Type:     schema.TypeString,
 										Required: true,
 										ValidateFunc: validation.StringInSlice([]string{
+											"Microsoft.BareMetal/AzureVMware",
+											"Microsoft.BareMetal/CrayServers",
 											"Microsoft.Batch/batchAccounts",
 											"Microsoft.ContainerInstance/containerGroups",
+											"Microsoft.Databricks/workspaces"
```

Comment:
from Travis - looks like there's a missing comma here:

```suggestion
											"Microsoft.Databricks/workspaces",
```

other than that this should be good to merge 👍

Reply:
fixed in new commit 8573366 which is now part of the PR.